test/fixtures/devtools-extension/content.js

```diff
@@ -1,4 +1,6 @@
 test("devtools.inspectedWindow.eval resolved with an error result", async (t) => {
+  t.timeoutAfter(5000);
```

Comment:
This won't work. If you look at the implementation that I linked from my previous comment, then you'll see that `timeoutAfter` is implemented by installing a non-cancelable timer that times out the test. Since our `test` wrapper already calls `timeoutAfter`, the only way to increase the timeout beyond 500ms is to update the `test` wrapper, using the instructions from my comment.